src/gui/redux/docgen/docgen.selectors.js

```diff
@@ -1,15 +1,12 @@
 import { createSelector } from 'reselect'
+import docgen from './docgen'
 
-const getDocgen = state => state.docgen
+const getDocgen = state => docgen.get()(state) || {}
```

Comment:
I need to do that, because if it's reseted the `state.data` will be `undefined`

Maybe you could add an optional property in the factory to initalize the data : 
```es6
factory()()()({ name: 'docgen', type: 'uniq', defaultData: {} })
```